### DISCUSSION TITLE
你们怎么知道我免费十连出薇塔？

### DISCUSSION THREAD

**Discussion by emmGrinch** [share121/inter-knot#5223] [崩坏3]
![IMG_20240920_042138](https://github.com/user-attachments/assets/7aa21188-b9d3-4e2e-829f-450f4c33762f)

**Comment by Lanxey** [comment]
知子莫若父

**Reply by emmGrinch** [reply]
<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >唉，总有人想超级加辈

**Comment by shirekemi** [comment]
新进来的给这位点点举报

**Reply by emmGrinch** [reply]
？真不行吧<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/40f465ef5312e4ba0b19c596b9999cef_8760502457253372865.png" alt="三月七 无语" width="75" >

**Comment by Anslate** [comment]
举办了🙃

**Reply by emmGrinch** [reply]
？我要红温了<img src="https://img-static.mihoyo.com/communityweb/upload/677e18698f28c655cd39bfb7f16a86f2.png" alt="卡芙卡 怒" width="75" >

**Comment by funnyyyyyyyy** [comment]
谁问你了？

**Reply by emmGrinch** [reply]
我问了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by qwertyuiop705** [comment]
打洗你！！！

**Reply by emmGrinch** [reply]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/a7304904a31329464114b1074218967e.png" alt="帕姆 瘫倒" width="75" >唉

**Comment by flystarnx** [comment]
三爷：俺也一样<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/829f0a1d533765b80769dd1a38dc338c_9182108590963360703.png" alt="阿姬惊讶" width="75" >

**Comment by 123456789question** [comment]
谁问你了喂

**Comment by Rjjfk** [comment]
![Screenshot_20240921_235335_com miHoYo enterprise NGHSoD](https://github.com/user-attachments/assets/f7aef51f-194a-4bdb-b887-216b037ab705)
不要再秀了啊（哭）bushi

**Comment by 596wjr** [comment]
你怎么知道我免费十连出希娜

**Comment by qwertyuiop705** [comment]
哇，免费十连出希娜
![1724907659046f1a3d8c765778dcace358fecfee8fbe378a0a69ab2547e7d34469df8da8bf165 0](https://github.com/user-attachments/assets/ce4ac07d-39da-4c80-87c5-c53170d300a0)

**Comment by Lephaethon** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/07/31/33f69e7df0fb9d06592379f11d7ba6ba_1797154502027561525.png" alt="云璃 威慑" width="75" >
谁问你了？？？

**Comment by nabeilishi** [comment]
谁问你了？

**Comment by Butt-Shark** [comment]
不玩崩三，过

**Reply by emmGrinch** [reply]
？我明明发在崩三版块😡

**Comment by xiangzhexch** [comment]
你不许玩崩坏三QAQ

**Reply by emmGrinch** [reply]
就玩，好玩，爱玩🤗<img src="https://bbs-static.miyoushe.com/static/2024/09/10/ed624befad4ed330fbdb41c91fc09d10_3535401542533232063.png" alt="飞霄 夸夸" width="75" >

**Comment by CRP123456** [comment]
谁问你了😡

**Reply by emmGrinch** [reply]
我问了😡

**Comment by aomengxin** [comment]
爪巴

**Reply by aomengxin** [reply]
我兄弟看不得这个
